JsonLogObject.php

```diff
@@ -84,6 +84,22 @@ public function __toString()
     {
         $this->values['timestamp'] = $this->makeTimestamp(microtime());
 
-        return json_encode($this->values);
+        return json_encode($this->ensureUtf8($this->values));
+    }
+
+    private function ensureUtf8($element) {
```

Comment:
The name of the new function is deceiving - can it be changed?
The function will only transform ISO-8859-1 and Windows-1252 characters to UTF-8. Any other encoding, including UTF-8, will be garbled if passed through this function.